concourse/tasks/delete_admin.yml

```diff
@@ -3,32 +3,46 @@ platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: governmentpaas/cf-uaac
+    repository: governmentpaas/cf-cli
+    tag: feature_126965751_move_cf_cli_to_alpine
 inputs:
   - name: paas-cf
   - name: cf-secrets
   - name: cf-manifest
   - name: admin-creds
   - name: bosh-CA
+  - name: config
 run:
   path: sh
   args:
     - -e
+    - -u
     - -c
     - |
-      NAME=$(cat admin-creds/username)
-      if [ "${ENABLE_ADMIN_USER_CREATION}" == "false" ]; then
+      if [ "${ENABLE_ADMIN_USER_CREATION:-}" == "false" ]; then
         echo "Temporary user creation is disabled (ENABLE_ADMIN_USER_CREATION=${ENABLE_ADMIN_USER_CREATION}). Skipping."
         exit 0
       fi
       ./paas-cf/concourse/scripts/import_bosh_ca.sh
+      . ./config/config.sh
 
-      VAL_FROM_YAML=$(pwd)/paas-cf/concourse/scripts/val_from_yaml.rb
-      UAA_ADMIN_CLIENT_PASS=$($VAL_FROM_YAML secrets.uaa_admin_client_secret cf-secrets/cf-secrets.yml)
-      UAA_ENDPOINT=$($VAL_FROM_YAML properties.uaa.url cf-manifest/cf-manifest.yml)
+      NAME=$(cat admin-creds/username)
 
       echo "Removing user ${NAME}"
-      uaac target "${UAA_ENDPOINT}"
-      uaac token client get admin -s "${UAA_ADMIN_CLIENT_PASS}"
 
-      uaac user delete "${NAME}"
+      echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+
+      cf delete-user "${NAME}" -f
+
+      echo "Removing leaked ${PREFIX} users"
```

Comment:
Personally, I would keep it, because the delete user can always fail because some random reason. 

The question then is why we do delete the user created explicitly , when we will delete it anyway when deleting all the tmp users.